frontend/src/cards/TableCard.tsx

```diff
@@ -139,8 +139,8 @@ export function TableCard(props: TableCardProps) {
                     (colName) => !NEVER_SHOW_PROPERTIES.includes(colName)
                   )}
                   variableId={props.variableConfig.variableId}
-                  variableName={props.variableConfig.variableFullDisplayName}
                   fips={props.fips}
+                  dataTableTitle={props.variableConfig.dataTableTitle ?? ""}
```

Comment:
Maybe this should default to `Data` or something generic so that the title never gets weird and unreadable if we forget to fill in the optional metric config value